src/databricks/labs/ucx/azure/resources.py

```diff
@@ -94,6 +115,18 @@ class AzureRoleAssignment:
     role_name: str
 
 
+@dataclass
+class AccessConnector:
```

Comment:
and you probably want to isolate the `RawResource -> AccessConnector` conversion to a single method.